### PR TITLE
fix: issue#31: checks if the width and height are negative and returns 0 if so, otherwise returns calculated values

### PR DESCRIPTION
**Pull request by AlanLes**
# Motivation

Regarding issue #31

Fixes [#31](https://github.com/Hexastack/eazychart/issues/31)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have done the work for both react and vue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (Storybook)
- [x] I have added unit tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/ez-react/src/components/Chart.tsx [hunk 1]
```diff
@@ -81,10 +81,12 @@ export const Chart: FC<ChartProps> = ({
...
       : 0;
-    // It take the container dimensions and subsctract padding and legend height
+    // It takes the container dimensions and subtracts padding and legend height
+    const width = containerDimensions.width - left - right;
+    const height = containerDimensions.height - top - bottom - legendHeight;
     return {
-      width: containerDimensions.width - left - right,
-      height: containerDimensions.height - top - bottom - legendHeight,
+      width: width >= 0 ? width : 0,
+      height: height >= 0 ? height : 0,
```
Comment (marrouchi) [inline packages/ez-react/src/components/Chart.tsx:89]:
```suggestion
      width: Math.max(width, 0),
      height: Math.max(height, 0),
```
Mind updating this also in vue ?

**Review by marrouchi — CHANGES_REQUESTED** [review]
@AlanLes Many thanks for your contribution ! :pray:

**Comment by AlanLes** [comment]
@marrouchi, I have some issues with committing, I can't commit changes, any ideas about what's going on? 🤔 😞 
image

**Comment by marrouchi** [comment]
> @marrouchi, I have some issues with committing, I can't commit changes, any ideas about what's going on? thinking disappointed image

@AlanLes Once you cloned the project did you run the command `yarn setup` ? This should install all dependencies.

**Comment by AlanLes** [comment]
Good point, huge thanks, it's finished 👌 🙂

File: packages/ez-react/src/components/Chart.tsx (hunk 1 above)
Comment (AlanLes) [reply packages/ez-react/src/components/Chart.tsx]: Okay, it's done! ;)

**Review by marrouchi — APPROVED** [review]
Awesome !

**Pull request merged**